src/main/java/CarGameStarter.java

```diff
@@ -0,0 +1,43 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+class CarGameStarter {
+  private List<String> carNames = new ArrayList<>();
+  private int trialNum;
+
+  CarGame startGame() {
+    getCarNamesFromUser();
+    getNumOfTrialFromUser();
+    List<Car> cars = carNames.stream()
+        .map(carName -> new Car(carName))
```

Comment:
```suggestion
        .map(Car::new)
```
람다 레퍼런스를 쓰면 조금 더 깔끔해질 수 있습니다!